client/src/pages/learn/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/index.md

```diff
@@ -5,6 +5,13 @@ superBlock: Information Security and Quality Assurance
 ---
 ## Introduction to the Information Security and Quality Assurance Projects
 
-This introduction is a stub
+So, now it’s time to put your newly learnt skills to work! By working on these projects, you will have the opportunity of applying all the skills, principles, and concepts from the Information Security and Quality Assurance sections.
 
-Help us make it real on [GitHub](https://github.com/freeCodeCamp/learn/tree/master/src/introductions).
\ No newline at end of file
+In this section you get the chance to:
+* Build a Metric-Imperial Converter
+* Build an Issue Tracker
+* Build a Personal Library
+* Build a Stock Price Checker
+* Build an Anonymous Message Board
+
+When you are done, you will have five Information Security & Quality Assurance projects under your belt, along with a certification, that you can show off to friends, family, and employers. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.
```

Comment:
Do we want to list the number of challenges over here>

cc: @RandellDawson @moT01

Reply:
It says there's five projects, I don't like to be repetitive - what did you have in mind?

Reply:
I want to get the number removed completely. See https://github.com/freeCodeCamp/freeCodeCamp/pull/34405#discussion_r264282065

Reply:
Sounds good to me either way, but I suppose that's a good point - if the number ever changes, we don't have to modify these.

Reply:
@Sumer16 Go ahead and reword this so it doesn't show no of challenges